packages/jest-each/src/bind.ts

```diff
@@ -26,17 +26,21 @@ type GlobalCallback = (
   testName: string,
   fn: Global.ConcurrentTestFn,
   timeout?: number,
+  eachError?: Error,
 ) => void;
 
 export default function bind<EachCallback extends Global.TestCallback>(
   cb: GlobalCallback,
   supportsDone = true,
-) {
-  return (
+  needsEachError = false,
+): Global.EachTestFn<any> {
```

Comment:
```suggestion
): Global.EachTestFn<unknown> {
```

Reply:
ts seems to complain, I don't understand why.